docs/source/langevin_sde.rst

```diff
@@ -0,0 +1,26 @@
+Stochastic Differential Equations (SDE) for Langevin dynamics
+=============================================================
+
+The general Langevin dynamics can be described by the following SDE:
+
+.. math::
+
+    d c_{i} = -\lambda\partial_{i} f(\mathbf{c}) + \sigma dW_{i}\;\;\text{with}\;\;c_{i}(0)=0\;\;\forall i=1,\ldots,N
+
+where :math:`\lambda` and :math:`\sigma` are hyper-parameters controlling the strengths of the drift and diffusion terms, respectively. We have implemented this solver as a classical solver implemented solely on classical computer, that is, there is no optical simulation of the Langevin dynamics in our ``ccvm`` package.
+
+Example script to solve BoxQP problem using Langevin solver
+-------------------------------------------------------------------
+
+On the command line, if you run the demo script by :code:`$ python ccvm/examples/langevin_boxqp.py`, it will simulate Langevin dynamics for an example problem instance and then the outcome is displayed as:
```

Comment:
Thanks @KatieO4t1qbit, it looks much better.

Reply:
@KatieO4t1qbit:
Line 12: Example script to solve BoxQP problem using Langevin solver

1. A matter of grammar: "to solve" should read "for solving".

2. As this documentation is new to me except for having reviewed the main README file and other very small, related bits, I'm wondering if all of your titles are this telegraphic in nature (no articles are used, I mean). If they are, that's fine. Otherwise, we'd have "Example script for solving **_a_** BoxQP problem using **_a/the_** Langevin solver"   (I'm not sure if we are talking about a particular Langevin solver or Langevin solvers in general, where we would use "the" or "a", respectively.)

Reply:
Thanks! Fixed both issues.